FirebaseAuthUI/FUIPasswordSignInViewController.m

```diff
@@ -134,7 +139,11 @@ - (void)signInWithDefaultValue:(NSString *)email andPassword:(NSString *)passwor
         }
       }
       [self.navigationController dismissViewControllerAnimated:YES completion:^{
-        [self.authUI invokeResultCallbackWithAuthDataResult:authResult error:error];
+        if (self->_onDismissCallback) {
+          self->_onDismissCallback(authResult, error);
```

Comment:
It is usually a good idea to nil-out callback blocks after use to avoid circular references.

Reply:
I went with the property approach outlined in you other comment. Since the block will be reused I leave it up to the developer using a custom block to decide whether or not it should be set to nil.